src/contexts/Maci.tsx

```diff
@@ -52,9 +52,14 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }) => {
 
   const attestationId = useMemo(() => {
     const values = attestations.data?.valueOf() as Attestation[] | undefined;
-    const length = values?.length ?? 1;
 
-    return values?.[length - 1]?.id;
+    let ret
+    values?.forEach((attestation) => {
+      if (config.admins.indexOf(attestation.attester) !== -1) 
+        ret = attestation.id;
```

Comment:
could this break out early if it finds the valid attester?

Reply:
I think forEach in typescript is not breakable: https://stackoverflow.com/questions/51747397/how-to-break-foreach-loop-in-typescript

But I also consider to use for loop instead of forEach, not sure which is better.

Reply:
Probably for loop so you can return directly once found?